src/main/resources/templates/fragments.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org/">
+
+<head th:fragment="head">
+
+    <!-- Bootstrap stylesheets and script -->
+    <link th:href="@{/css/bootstrap.css}" rel="stylesheet" />
```

Comment:
I suggest waiting until @gerarbot-js merges his bootstrap PR and then refactoring it to use that technique to pull in bootstrap.